go.mod

```diff
@@ -4,6 +4,7 @@ go 1.14
 
 require (
 	github.com/ghodss/yaml v1.0.0
-	github.com/golang/protobuf v1.3.5
+	github.com/golang/protobuf v1.4.0-rc.4
```

Comment:
Wait, do we now need both `github.com/golang/protobuf` and `google.golang.org/protobuf v1.20.1`?

Reply:
Yeah, that was surprising to me at first, but the [blog post](https://blog.golang.org/protobuf-apiv2) says that:
> google.golang.org/protobuf@v1.20.0 is APIv2. This module depends upon github.com/golang/protobuf@v1.4.0, so any program which uses APIv2 will automatically pick a version of APIv1 which integrates with it.

So I'm pretty sure this is expected if I'm reading that correctly.

Reply:
👍 🤷‍♀

Reply:
Ah, here's some more context:
https://github.com/protocolbuffers/protobuf-go/commit/baf64d5536c1219775e356cc42c08346524e8e68

Reply:
Uh oh, I spy an unassigned TODO in that commit